### ISSUE TITLE
Bug: "Trying to access array offset on value of type null"

### ISSUE THREAD

**Issue by Tagirijus** [xenocrat/chyrp-lite#173]
Hey there,

with the new version _2022.03_ it seems like I cannot visit the _home_, _archive_ or _feed_ site. It gives me the error:

```
Trying to access array offset on value of type null
```

I did every step mentioned under _Upgrade_ in the README.md. What else can I do to give a proper debug info for you? Or do you maybe already can reproduce it on your end (magically)? :D

**Comment by xenocrat** [comment]
Hello there,

Please create a file named DEBUG in your _includes_ folder. With this file present, the error should then be reported with a backtrace. If you copy and past the backtrace I can investigate further. As much additional information as possible is also helpful: PHP version, platform (Linux / Mac / Windows etc.), database (MySQL, SQLite, PostgreSQL, clean install or upgrade?

**Comment by Tagirijus** [comment]
Oh sorry, this even is described [here in the wiki](https://chyrplite.net/wiki/Debug-and-Tester-Modes.html) which I did not check. Thanks for the info.

I now switched forth and back, re-checked the `config.json.php` and even tried a fresh install. Suddenly the fresh install of the older version did also not work. I must have broke something important without noticing or so.

**But:** Now it is working again as it seems. There might have been something with the `.htaccess`, but I am not sure. In the end I am so sorry that I did not keep track of all the things I just did. But it is working again now.

If something else will be weird again, I now know how to enable the debug mode. Thanks a lot for that information.

Again: sorry for bothering, argh!
